dashboard/config/environments/test.rb

```diff
@@ -39,8 +39,11 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
-  # Whether or not to display pretty blockly.
-  config.pretty_apps = false
+  # Whether or not to display pretty apps (formerly called blockly).
+  config.pretty_apps = true
```

Comment:
should we keep these settings on `test` in sync with `production`, since we're still using this environment on our pre-prod server? If we set these to true, we won't shake out any hiccups in the minified build process until the files go live on production.